languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/remote-rule-filters.xml

```diff
@@ -1029,6 +1029,20 @@ To ignore a remote rule match, set the <marker> so that it exactly covers the te
                 <example correction="">Einer beziehung zerstörst, was sie bequem für lust statt bars zu versuchen sie mag, <marker>das</marker>.</example>
             </rule>
         </rulegroup>
+        <rulegroup id="AI_DE_HYDRA_LEO_CP_DEM_DEN" name="">
+            <rule>
+                <pattern>
+                    <token>in</token>
+                    <marker>
+                        <token>dem</token>
+                    </marker>
+                    <token postag="SUB:DAT:SIN:NEU">
```

Comment:
sieht ok aus, ich versuche nur zu überlegen, ob es cases gibt, in denen die Regel richtig anschlägt und die durch das AP verhindert werden könnten. aber mir fällt momentan nix ein